include/uriparser/Uri.h

```diff
@@ -44,6 +44,8 @@
  * NOTE: This header includes itself twice.
  */
 
+#include "uriparser_export.h"
```

Comment:
Is that file generated on the fly? Is it missing from the pull request?

Reply:
Yes. It is generated on the fly.